WordPress/Classes/Utility/FormattableContent/FormattableContent.swift

```diff
@@ -36,4 +36,15 @@ extension FormattableContent {
             $0.identifier == id
         }.first
     }
+
+    func range(with url: URL) -> FormattableContentRange? {
+        let linkRanges = ranges.compactMap { $0 as? LinkContentRange }
+        for range in linkRanges {
```

Comment:
Would it make sense to filter the linkRanges using `.filter` instead of looping them?

Reply:
One benefit from `for in` is the early exit as soon as it finds a match. `.filter` will continue looping even if it already found the match.
I checked if the syntax is cleaner but they are quite similar.

I'm inclined to leave it as it is unless I'm missing something. 😮